kbs/config/kubernetes/README.md

```diff
@@ -9,7 +9,7 @@ We will see how to deploy KBS (with builtin Attestation Service) on a Kubernetes
 Create a secret that you want to be served using this instance of KBS:
 
 ```bash
-echo "This is my super secert" > overlays/key.bin
+echo "This is my super secert" > overlays/$(uname -m)/key.bin
```

Comment:
Should probably fix `secert` -> `secret`

Reply:
Yep, I will change it. Thanks!